src/marqo/tensor_search/tensor_search.py

```diff
@@ -81,6 +81,47 @@
 logger = get_logger(__name__)
 
 
+def _get_dimension_from_model_properties(model_properties: dict):
```

Comment:
note for future work: this, and the other `get_model_properties()` function can be moved to its file, for example: `models/model_properties.py`